### ISSUE TITLE
Account moderation list visibility issue on mobile since v4.3.4

### ISSUE THREAD

**Issue by DasMetalKitty** [mastodon/mastodon#34030]
### Steps to reproduce the problem

1.Since the upgrade to v4.3.4 the mobile screen display of the moderation module for accounts has a truncated display and does not conform with WCAG accessibility standards. 
The details of account user is no longer displayed in portrait orientation of the screen. Which means scanning the list of accounts is now no longer possible this way. When landscape orientation is used, the table opens up as previous to the upgrade but now only displays 3 accounts at a time instead of the ability to see 8 at once.

This situation has been checked and confirmed with 2 other Mastodon admins on their respective mobiles too.

Please revert to the old table visibility so Mastodon admins and moderators can use the tools on their mobiles instead of being limited to bigger devices such as laptops. Thank you

...

### Expected behaviour

As described above in first field

### Actual behaviour

As described above in first field

### Mastodon instance

mastoart.social and on metalhead.club

### Mastodon version

v3.4.3

### Browser name and version

3 different users, at least one using FireFox

### Operating system

3 different mobile phones at least one Android mobile

**Comment by ThomasLeister** [comment]
Adding a picture of the issue: 

![Image](https://github.com/user-attachments/assets/fcabe070-7ce1-4f89-bc92-b98e44a05bfb)

The usernames cannot be read on this page: /admin/accounts?origin=local
... which is quite unpleasing given that I do most moderation work on my mobile phone ;-)

**Comment by ThomasLeister** [comment]
Adding a picture of the issue:

![Image](https://github.com/user-attachments/assets/5daf94fe-9143-4254-bf18-994d19ed4444)

The usernames cannot be read on this page: /admin/accounts?origin=local
... which is quite unpleasing given that I do most moderation work on my mobile phone ;-)

**Comment by Sciasm** [comment]
Can confirm this on a Google Pixel 7a (Android 15).

Being able to see usernames played a crucial part in identifying remote spam accounts. Please kindly consider reverting the truncation of usernames.

**Comment by ClearlyClaire** [comment]
Seems like this was introduced in #29316, which I had flagged for backporting. I had since (I believe) addressed the issue in #33064, but I skipped backporting it.

**Comment by ClearlyClaire** [comment]
This should be fixed in 4.3.5.
